DependencyInjection/CompilerPass/ImageFormatCompilerPass.php

```diff
@@ -36,30 +37,11 @@ class ImageFormatCompilerPass implements CompilerPassInterface
      */
     public function process(ContainerBuilder $container)
     {
-        $this->container = $container;
-
-        $formats = $this->loadThemeFormats(
-            $container->getParameter('sulu_media.format_manager.default_imagine_options')
-        );
-        if ($container->hasParameter('sulu_media.image.formats')) {
-            $formats = array_merge($container->getParameter('sulu_media.image.formats'), $formats);
-        }
-
-        $container->setParameter('sulu_media.image.formats', $formats);
-    }
-
-    /**
-     * @param array $globalOptions
-     *
-     * @return array
-     */
-    private function loadThemeFormats($globalOptions)
-    {
-        $activeFormats = [];
-        $activeTheme = $this->container->get('liip_theme.active_theme');
-        $bundles = $this->container->getParameter('kernel.bundles');
-        $configPaths = $this->container->getParameter('sulu_media.format_manager.config_paths');
```

Comment:
Not sure what I should do about this parameter. For now I have just haven't used it in this class. IMHO it's strange that this configuration is in the `sulu_media` namespace, and that this is the only usage from it.

It should be quite safe to remove, because I can't imagine anybody used it and it was also not documented anywhere. In case you think different we can reintroduce it, but then I would do it in the `sulu_theme` namespace.

If you agree I would also remove the parameter in https://github.com/sulu/sulu/pull/3040.

/cc @chirimoya @wachterjohannes @alexander-schranz